test/Microsoft.Extensions.Configuration.FileExtensions.Test/FileConfigurationBuilderExtensionsTest.cs

```diff
@@ -54,13 +54,12 @@ public void GetBasePath_ReturnBaseDirectoryIfNotSet()
             // Act
             var actualPath = configurationBuilder.GetBasePath();
 
-            string expectedPath = string.Empty;
+            string expectedPath;
 
 #if DNXCORE50
-             expectedPath = AppContext.BaseDirectory;
+            expectedPath = AppContext.BaseDirectory;
 #else
-            expectedPath = AppDomain.CurrentDomain.GetData("APP_CONTEXT_BASE_DIRECTORY") as string ??
-                AppDomain.CurrentDomain.BaseDirectory ?? string.Empty;
```

Comment:
Yeah - the test passes using `APP_CONTEXT_BASE_DIRECTORY`. It'd be useful to know when it breaks rather than verify we have a mirror of the code in `src`.